.github/actions/rust/action.yaml

```diff
@@ -32,8 +32,15 @@ runs:
         echo "PROTOC=${HOME}/.local/bin/protoc" >> $GITHUB_ENV
         export PATH="${PATH}:${HOME}/.local/bin"
 
+    - name: Configure AWS credentials and bucket region
+      uses: aws-actions/configure-aws-credentials@v4
+      if: contains(runner.name, 'ubuntu-platform')
+      with:
+        aws-region: eu-west-1
+
     - name: Cache Rust build artifacts
```

Comment:
Let's add action/cache otherwise (`!contains(runner.name, 'ubuntu-platform')`)?

Reply:
Ah now I understand what you mean! Yes good idea, I've added GHA caching with suitable conditions